pkg/appfile/appfile.go

```diff
@@ -17,7 +17,7 @@ var (
 	ErrImageNotDefined = errors.New("image not defined")
 )
 
-const DefaultAppfilePath = "./vela.yml"
+const DefaultAppfilePath = "./vela.yaml"
```

Comment:
let's support both of them

Reply:
check vela.yaml first and then vela.yml

Reply:
Let's use `.yaml` only. It's rare and weird for a tool support both.

Reply:
I'm OK with it。